pkg/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -4,17 +4,16 @@ import (
 	"bytes"
 	"fmt"
 	"io/ioutil"
-	"sort"
 	"strconv"
 	"strings"
 	"text/template"
 
 	performancev1alpha1 "github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1alpha1"
 	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components"
+	profile2 "github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components/profile"
```

Comment:
minor: `profile2` reminds of a version, and that's a bit misleading. If you called like this only to avoid name clashes, maybe we can use something like `perfprofile` or `compprofile` ?